requirements/customer/scheduling-algorithm.tex

```diff
@@ -63,4 +63,28 @@ \section{Scheduling Algorithm}
   By default, the scheduling algorithm will \textbf{not} pause running jobs to free up resources to run non-urgent jobs.
   However, this behavior changes once the effective priority $p$ of a non-urgent job reaches a certain threshold:
   once the threshold is reached the scheduling algorithm will begin to pause non-urgent running jobs with lower effective priority $p$ if possible and necessary to run the job with an effective priority $p$ above the threshold.
-\end{enumerate}
\ No newline at end of file
+\end{enumerate}
+\begin{figure}[H]
+  \centering
+  \includegraphics[scale=0.5]{./customer/diagrams/png/main-idea.png}
+  \caption{This diagram depicts the main flow of the scheduling algorithm}
+\end{figure}
+\begin{figure}[H]
+  \centering
+  \includegraphics[scale=0.5]{./customer/diagrams/png/urgent-job.png}
+  \caption{This diagram depicts how the algorithm process an urgent job.
+  This process repeats itself for every pending urgent job, starting with the lowest effective priority one.}
+\end{figure}
+\begin{figure}[H]
+  \centering
+  \includegraphics[scale=0.5]{./customer/diagrams/png/paused-job.png}
```

Comment:
Paused job -> paused non-urgent job
Can the job resume running -> can the job be resumed
Are there jobs with lower priority running on the work machine -> Are there **non-urgent** jobs with lower priority running on the work machine